clients/filesystem-hadoop/src/main/java/com/datastrato/gravitino/filesystem/hadoop/GravitinoVirtualFileSystemConfiguration.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.filesystem.hadoop;
+
+public class GravitinoVirtualFileSystemConfiguration {
+  public static final String GVFS_FILESET_PREFIX = "gvfs://fileset";
+  public static final String GVFS_SCHEME = "gvfs";
+  public static final String HDFS_SCHEME_PREFIX = "hdfs://";
+  public static final String FS_GRAVITINO_SERVER_URI_KEY = "fs.gravitino.server.uri";
```

Comment:
Should we follow the style `fs.%s.impl.xxxx`?

Reply:
I think it is not necessary, refer to the parameters of hadoop azure: https://hadoop.apache.org/docs/current/hadoop-azure/index.html

Reply:
OK.